.storybook/addons.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
```

Comment:
Can we not use a `/examples/<any>` here? Do we really have to add something to the root of the project?

Reply:
I would be happy with an `examples/official` or something, where we have a storybook to demo storybook's UI components.

Reply:
I don't have any problem with that. The only thing is potentially wanting to combine all react-based storybooks in a single storybook (components, ui, cra-vanilla).

[This works better for chromatic for instance. I'm not sure it helps much else -- maybe storyshots?]

Maybe `examples/offical-storybook` or something?

Reply:
Or `examples/meta[-storybook]`?

Reply:
`examples/offical-storybook` would get my vote